src/modules/PostService.js

```diff
@@ -28,6 +28,24 @@ const PostService = {
       })
     }
   },
+  async delete(postId, credentials) {
+    try {
+      let response = await axios.delete(
+        `${API_URL}/posts/${postId}`,
+        { post_id: postId },
+        { headers: credentials }
```

Comment:
we're not accepting credentials in the backend, so we can probably remove it here

Reply:
Yes! Got it! Thanks.